tests/test_views.py

```diff
@@ -45,7 +45,7 @@ def test_airtable_message_on_instance_create(self):
         })
```

Comment:
Fixes in this file are because of failing tests.  
It might be worth considering addind linting in order to avoid this in the future